Codeinwp/templates-patterns-collection
# Chore/move onboarding

### Summary
Moves the onboarding UI from the Neve Dashboard to this plugin in its own admin page.


### Test instructions


- This should work as a standalone. It will require and install/activate Neve when you try importing though.
- Admin page should work as expected.


REF Codeinwp/neve-pro-addon#854.
